app/views/spree/shared/_search.html.erb

```diff
@@ -14,7 +14,7 @@
   </button> -->
 <%# end %>
 <%= form_tag spree.products_path, method: :get , id:'global-search-button' do%>
-  <div class="header-search tablet-hide" id="search-dropdown">
```

Comment:
verify why we delete this id

Reply:
sir it was my mistake to add this id from previous spree because it was nowhere in current spree and this issue was pointed by umer bhai